inference-engine/tests/functional/plugin/gpu/remote_ctx_tests/dx11_remote_ctx_test.cpp

```diff
@@ -0,0 +1,134 @@
+// Copyright (C) 2021 Intel Corporation
```

Comment:
we already have folder for remote tests: `inference-engine/tests/functional/plugin/gpu/remote_blob_tests`. Let's put this file under the same folder

Reply:
If you don't like `blob` word in the folder name, feel free to rename the folder to something like `remote_api_tests`